### ISSUE TITLE
Please make the Driver public

### ISSUE THREAD

**Issue by mikeblakeuk** [endjin/Endjin.SpecFlow.Selenium#1]
There are times that I need access to the Selenium Driver (e.g. `Driver.SwitchTo().Alert()`)
Please can you make **Driver** a read-only property of the Navigator. Thanks

Workaround

``` c#
var d = typeof(Navigator).GetField("driver", BindingFlags.NonPublic | BindingFlags.Instance);
var v = d.GetValue(Navigator.Browser);
return (EventFiringWebDriver)v;
```

**Comment by HowardvanRooijen** [comment]
OK - that shouldn't be too difficult.

**Comment by AliceWaddicor** [comment]
Fixed in release 2.0.0.1
